src/main/java/edu/kis/powp/jobs2d/command/ComplexCommandUsage.java

```diff
@@ -0,0 +1,30 @@
+package edu.kis.powp.jobs2d.command;
+
+public class ComplexCommandUsage {
+
+
+    public static ComplexCommand drawRectangle(int x, int y, int off1, int off2)
```

Comment:
the method does not "draw" it returns command instance.
Is it important to have ComplexCommand return type?